.storybook/config.js

```diff
@@ -14,7 +14,7 @@ const ThemesDecorator = (storyFn) => (
         .storyWrapperLight {
           width: 400px;
           padding: 40px;
-          background: ${defaultTheme.window.backgroundColor};
```

Comment:
If we're not using `defaultTheme` and `darkTheme` here anymore we can remove imports as well